app/bounty_requests/views.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+"""Define bounty request views.
+
+Copyright (C) 2018 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+import json
+
+from django.http import JsonResponse
+from django.template.response import TemplateResponse
+from django.utils.html import escape, strip_tags
+from django.utils.translation import gettext_lazy as _
+from django.views.decorators.csrf import csrf_exempt
+
+from marketing.mails import new_bounty_request
+from ratelimit.decorators import ratelimit
+
+from .models import BountyRequest
+
+
+@ratelimit(key='ip', rate='5/m', method=ratelimit.UNSAFE, block=True)
+@csrf_exempt
+def bounty_request(request):
+    user = request.user if request.user.is_authenticated else None
+    profile = request.user.profile if user and hasattr(request.user, 'profile') else None
+
+    if request.body:
```

Comment:
I can take a look into django forms. Have not worked with them before. Or django at all 😆 .
A decided not to use excessive validation because it would ultimately fail at model creation if something is really wrong, and it doesn't really harm throwing 500s for V0 if someone plays with the endpoint.

However, you decide 😄 . Let me know 👍

Reply:
I would agree with @mbeacom . The idea is @pinkiebell if you leave something in v0, it later becomes a tech debt and debts always keep growing. 😅 I would really encourage you to have a look and use django forms. If you find any difficulty in implementing or understanding something, feel free to ping me (or @mbeacom).

Reply:
drf looks more promising. Could be done in a separate issue, maybe even for the entire project?

Reply:
@SaptakS Oh, didn't saw your comment.
I will dive into django forms 👍

Reply:
We definitely need to work on migrating all of our views to utilize forms where possible.  But frankly, 🕐 is the issue here. I'm not opposed to tip-for-turnaround or bountying it out in the meantime while we work towards getting in a position to make the adjustments internally.

As for DRF - It only makes sense when the situation dictates it. Forms would be a good idea in the meantime, until we can establish a good example for other community contributors to work off of (as it relates to integrating DRF and forms appropriately).

Reply:
@pinkiebell That said... I'm fine with using your view as it is now, because I know @vs77bb is eager to get v0 out the door.  Moving forward, we'll update the community guidelines to recommend using django forms and DRF when appropriate.  @SaptakS While I agree, we should ensure these changes are in before merging, we don't really have much justification to request community contributors to do it since the project doesn't have any good examples of how to implement forms. 😭 Maybe we can have an internal discussion about this and start iteration, so there are examples of how to migrating existing views and implement new ones.

Reply:
@mbeacom @SaptakS 
If future guidelines/design decisions are not final yet,
I would be more than happy to have a uniform api model.
Streamlined like django forms for db models, but with JSON.

For example
POST /api/v1/bounty/request < usual {payload..}
GET /api/v1/bounty/request > [{'name': 'requested_by', 'default_value':'', 'type': 'number', 'required': true},...] and so on.